processes/kneaddata.nf

```diff
@@ -12,21 +12,21 @@ process kneaddata {
     path human_genome
 
     output:
-    tuple val(sample), path("${sample}_kneaddata_paired_{1,2}.fastq.gz")
-    path "${sample}_kneaddata_unmatched_{1,2}.fastq.gz"
+    tuple val(sample), path("${sample}_kneaddata.fastq.gz")
```

Comment:
same

Reply:
this output should end up gzipped due to the end of the process (which does `gzip *.fastq`